### PR TITLE
Enabling nyamber-runner image to run dctest

### PR DESCRIPTION
**Pull request by yamatcha**
This PR includes these changes
- Add dctest bootstrap script
- Enable switching script by selecting configmap
- Divide manifest with its kind

Signed-off-by: yamatcha <soju-yamashita@cybozu.co.jp>

### CODE REVIEW

File: include-bird [hunk 1]
```diff
@@ -0,0 +1 @@
+path-include=/usr/share/doc/bird2/*
```
Comment (umezawatakeshi) [inline include-bird:1]: Where is this file referred from...?

Comment (yamatcha) [reply include-bird:1]: `/etc/dpkg/dpkg.cfg.d/excludes` filters out `/usr/share/doc`. 
Here including ` /usr/share/doc/bird2/*` enable to create `/usr/share/doc/bird2/examples/bird.conf` during `apt install bird2`.
ref: https://bird.network.cz/pipermail/bird-users/2019-December/014075.html

File: config/manager/pod_template.yaml [hunk 2]
```diff
@@ -6,3 +6,11 @@ spec:
...
     command:
       - "/entrypoint" 
+    volumeMounts:
+    - name: scripts
+      mountPath: /scripts
+  volumes:
+  - name: scripts
+    configMap:
+      name: scripts
+      defaultMode: 0777
```
Comment (umezawatakeshi) [inline config/manager/pod_template.yaml:16]: Is it required to be writable?

**Review by umezawatakeshi — APPROVED** [review]

**Review by umezawatakeshi — APPROVED** [review]

File: Dockerfile.runner [hunk 3]
```diff
@@ -21,9 +21,66 @@ RUN --mount=type=cache,target=/go/pkg/mod \
...
+            sudo \
+    && rm -rf /var/lib/apt/lists/* \
+    && curl -sSLf https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz | tar -C /usr/local -xzf - \
+    && curl -sfL https://github.com/cybozu-go/placemat/releases/download/v${PLACEMAT_VERSION}/placemat2_${PLACEMAT_VERSION}_amd64.deb -o placemat2_${PLACEMAT_VERSION}_amd64.deb \
+    && dpkg -i ./placemat2_${PLACEMAT_VERSION}_amd64.deb \
+    && rm ./placemat2_${PLACEMAT_VERSION}_amd64.deb \
+    && echo "nyamber ALL=(ALL) NOPASSWD: ALL" > /etc/sudoers \
+    && adduser --disabled-password --gecos "" --uid 10000 nyamber \
+    && mkdir -p /opt/hostedtoolcache \
+    && chown -R nyamber:nyamber /opt/hostedtoolcache \
```
Comment (kmdkuk) [inline Dockerfile.runner:80]:
```suggestion
```
No need this lines.

**Review by kmdkuk — APPROVED** [review]
LGTM

**Pull request merged**